api/pyproject.toml

```diff
@@ -8,36 +8,37 @@ authors = [
   {name = "Openverse Contributors", email = "openverse@wordpress.org"},
 ]
 
-requires-python = "==3.11.*"
+requires-python = "==3.12.*"
 dependencies = [
-  "adrf >=0.1.4, <0.2",
-  "aiohttp >=3.9.3, <4",
-  "aws-requests-auth >=0.4.3, <0.5",
-  "deepdiff >=6.7.1, <7",
-  "django >=4.2.11, <5",
-  "django-asgi-lifespan >=0.3.1, <0.4",
-  "django-cors-headers >=4.3.1, <5",
-  "django-migrations-git-conflicts >=1.0, <2",
-  "django-oauth-toolkit >=3.0.1, <4",
-  "django-redis >=5.4.0, <6",
-  "django-split-settings >=1.2.0, <2",
-  "django-structlog >=8.0.0, <9",
-  "django-tqdm >=1.3.1, <2",
-  "django-uuslug >=2.0.0, <3",
-  "djangorestframework >=3.14.0, <4",
-  "drf-spectacular >=0.27.1, <0.28",
-  "elasticsearch >=8.13.0, <9",
-  "elasticsearch-dsl >=8.12.0, <9",
-  "future >=0.18.3, <0.19",
-  "limit >=0.2.3, <0.3",
-  "pillow >=10.2.0, <11",
-  "psycopg >=3.1.18, <4",
-  "python-decouple >=3.8, <4",
-  "python-xmp-toolkit >=2.0.2, <3",
-  "sentry-sdk >=1.40.6, <2",
-  "uvicorn[standard] >=0.30, <0.31",
-  "openverse-attribution @ file:///${PROJECT_ROOT}/../packages/python/openverse-attribution",
-  "structlog-sentry>=2.1.0",
+    "adrf >=0.1.4, <0.2",
+    "aiohttp >=3.9.3, <4",
+    "aws-requests-auth >=0.4.3, <0.5",
+    "deepdiff >=6.7.1, <7",
+    "django >=4.2.11, <5",
+    "django-asgi-lifespan >=0.3.1, <0.4",
+    "django-cors-headers >=4.3.1, <5",
+    "django-migrations-git-conflicts >=1.0, <2",
+    "django-oauth-toolkit >=3.0.1, <4",
+    "django-redis >=5.4.0, <6",
+    "django-split-settings >=1.2.0, <2",
+    "django-structlog >=8.0.0, <9",
+    "django-tqdm >=1.3.1, <2",
+    "django-uuslug >=2.0.0, <3",
+    "djangorestframework >=3.14.0, <4",
+    "drf-spectacular >=0.27.1, <0.28",
+    "elasticsearch==8.13.0",
+    "elasticsearch-dsl >=8.12.0, <9",
+    "future >=0.18.3, <0.19",
+    "limit >=0.2.3, <0.3",
+    "pillow >=10.2.0, <11",
+    "psycopg >=3.1.18, <4",
+    "python-decouple >=3.8, <4",
+    "python-xmp-toolkit >=2.0.2, <3",
+    "sentry-sdk >=1.40.6, <2",
+    "uvicorn[standard] >=0.30, <0.31",
+    "openverse-attribution @ file:///${PROJECT_ROOT}/../packages/python/openverse-attribution",
+    "structlog-sentry>=2.1.0",
+    "pook==2.0.0",
```

Comment:
Shouldn't this be in the `dev-dependencies` section? Is it needed for core application code?

Reply:
Removed in [c9a8fcb](https://github.com/WordPress/openverse/pull/5095/commits/c9a8fcbcdcfbb9696f33787bc1a5c5decab5b93a)